api/tests/unit/application/users/index_test.js

```diff
@@ -255,4 +255,26 @@ describe('Unit | Router | user-router', () => {
       });
     });
   });
+
+  describe('GET /api/users/{id}/organizations', function() {
+    beforeEach(() => {
+      sinon.stub(userController, 'getOrganizations').returns('ok');
+      sinon.stub(securityController, 'checkRequestedUserIsAuthenticatedUser').callsFake((request, h) => h.response(true));
+      startServer();
+    });
+
+    it('should exist', () => {
```

Comment:
Je pense qu'on peut mieux nommer cette méthode, ex : `should return the current user organizations`. 

C'est là où tu te rends comptes qu'il y a peut-être un petit problème de design d'API, d'implémentation ou de régression. On vérifie que le `userId` en paramètre est le même que celui authentifié via le header de la requête. Quid du cas où c'est Pix Admin qui tente d'afficher et de gérer les organisations de l'utilisateur ? 

Remarque (qui n'a pas complètement à voir avec cette PR) : par ailleurs, est-ce qu'on ne devrait pas faire comme GitHub, qui catégorise les actions propre à l'utilisateur connecté dans un namespace `/api/user/organizations` (vs. `/api/users/{id}/organizations`).

Reply:
Salut, c'est une bonne remarque, mais pour ce cas particulier je te renvoie vers la fin du commentaire explicatif de la PR que voici :

> vu que PixBoard est officiellement retiré de la circulation au 1er janvier, la PR qui organisera le retrait de code lié à cette feature embarquera avec elle l'appel réseau supplémentaire. Je veux bien m'occuper du retrait de PixBoard en 2020.